submission/launch.sh

```diff
@@ -27,4 +27,5 @@ jq -n \
     '{stdout: $out, exit_code: $code,}' > out.json
 
 curl -X POST "${VMCK_CALLBACK_URL}" -d @out.json \
-     --header "Content-Type: application/json"
+     --header "Content-Type: application/json" \
+     --retry 20 --retry-connrefused
```

Comment:
Q: should we have a config file for this retry value?

Reply:
Sure, but it would have to get passed from `acs-interface` through `vmck` to this script. I think this script should actually live in `acs-interface` which will make it easier to change things. Let's leave it hardcoded for now.